easybuild/easyblocks/t/tensorflow.py

```diff
@@ -421,10 +421,10 @@ def setup_build_dirs(self):
     def configure_step(self):
         """Custom configuration procedure for TensorFlow."""
 
-        binutils_root = get_software_root('binutils')
-        if not binutils_root:
-            raise EasyBuildError("Failed to determine installation prefix for binutils")
-        self.binutils_bin_path = os.path.join(binutils_root, 'bin')
+        # determine location where binutils' ld command is installed
+        # note that this may be an RPATH wrapper script (when EasyBuild is configured with --rpath)
```

Comment:
This comment makes me nervous, what if they expect to find more than just `ld`?

Reply:
We can consider symlinking the other binutils tools in that some location (or in a different path, with `ld` a symlink to the wrapper script).

There seems to be no need for this though, it works as expected this way...

Reply:
Can't fully remember but isn't this path what gets passed to `-B` to the compiler? The tools are also in PATH so maybe it does indeed work or we don't noticed that it doesn't...